core/validation_schema.py

```diff
@@ -26,6 +26,7 @@
             "Organisation": "str",
         },
         "uniques": ["Programme ID"],
+        "unique_togethers": ("Programme ID", "Organisation"),
```

Comment:
We either need to make the constraint function accept a single composite_key per table like how you've defined the schema here or put these tuples in a list

Reply:
done

Reply:
opted to modify the operation